implementations/code_df/code_changes_git.py

```diff
@@ -67,21 +68,21 @@ def compute_timeseries(self, period='month'):
                 .agg('count')
 
             all_periods = pd.DataFrame(
-                            pd.date_range(self.since, self.until, freq='M'),
-                            columns=['Dates'])
+                pd.date_range(self.since, self.until, freq='M'),
+                columns=['Dates'])
             all_periods = pd.DataFrame(
                 [all_periods['Dates'].dt.year.rename('year'),
                  all_periods['Dates'].dt.month.rename("month")]).T
 
         elif period == 'week':
             timeseries_series = df['created_date'] \
                 .groupby([df['created_date'].dt.year.rename('year'),
-                          df['created_date'].dt.week.rename('week')])   \
+                          df['created_date'].dt.week.rename('week')]) \
                 .agg('count')
 
             all_periods = pd.DataFrame(
-                            pd.date_range(self.since, self.until, freq='W'),
-                            columns=['Dates'])
+                pd.date_range(self.since, self.until, freq='W'),
+                columns=['Dates'])
```

Comment:
I had run the code through a `flake8` test. Is there a specific reason you suggest this change? Thanks

Reply:
These have happened when I ran the formatter in pyCharm, @Polaris000

Reply:
OK Thanks!